README.md

```diff
@@ -64,15 +64,10 @@ your project's `CMakeLists.txt` file as follows:
     cmake_minimum_required(VERSION 3.0)
     project(example)
 
-    find_package(wave REQUIRED)
+    find_package(wave REQUIRED geometry matching)
 
-    # Add a target depending on all of libwave
     add_executable(example example.cpp)
-    target_link_libraries(example wave::wave)
-
-    # Add a target depending on certain libwave components
-    add_executable(example2 example2.cpp)
-    target_link_libraries(example2 wave::geometry wave::containers)
+    target_link_libraries(example wave::geometry wave::matching)
```

Comment:
wave::wave will still work if you want the whole thing, right?

Reply:
Yes. But it's not a good idea to use it, since now you don't know what components are included. Maybe it should be removed?

Reply:
Actually if you ask for REQUIRED components inf find_package, then yes you could link to `wave::wave` and be sure those components are included. Better to be specific anyway IMO, so you're not accidentally relying on something else.

Reply:
Okay that makes sense.